src/java/org/apache/cassandra/db/virtual/SystemViewsKeyspace.java

```diff
@@ -45,6 +50,13 @@ private SystemViewsKeyspace()
                                                               new ThreadPoolRowWalker(),
                                                               Metrics.allThreadPoolMetrics(),
                                                               ThreadPoolRow::new))
+                    .add(CollectionVirtualTableAdapter.create(VIRTUAL_VIEWS,
+                                                              COMPACTION_OPERATIONS_HISTORY,
+                                                              "Compaction operations that are currently " +
+                                                              "running or have been completed since the node start",
```

Comment:
@Mmuzaf `since the node started` or `since node's start`

Reply:
Fixed.